src/buildercore/fastly/vcl/main.vcl

```diff
@@ -61,6 +64,12 @@ sub vcl_miss {
 }
 
 sub vcl_deliver {
+  if (resp.status >= 500 && resp.status < 600 && stale.exists) {
+    unset req.http.Cookie;
```

Comment:
`Cookie` is unset in two places, so what are the two different use cases?

Reply:
I think they include it in `deliver` as well as you don't necessarily go through `fetch`.

Reply:
Thinking about it, probably don't need it here (assume `stale.exists` would always be `false` if you have a cookie).

Not sure how to test this behaviour at the minute.

Reply:
Can we add something like `req.http.Some-Standard-Elife-Debug-Header` and get it logged so that we see if this section is ever evaluated with live traffic?